login.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.querySelector(".emailform .innertext"); // 이메일 input
+  const passwordInput = document.querySelector(".pswdform .innertext"); // 비밀번호 input
+  const loginButton = document.querySelector(".login-btn"); // 로그인 버튼
+  const emailLabel = document.querySelector(".emailform"); // 이메일 label
+  const passwordLabel = document.querySelector(".pswdform"); // 비밀번호 label
+  // 이메일 검사
+  emailInput.addEventListener("blur", function () {
+      validateEmail();
+      toggleLoginButton();
+  });
+  // 비밀번호 검사
+  passwordInput.addEventListener("blur", function () {
+      validatePassword();
+      toggleLoginButton();
+  });
+  // 로그인 버튼 클릭 시 /items로 이동
+  loginButton.addEventListener("click", function (event) {
+      if (loginButton.disabled) {
+          event.preventDefault(); // 버튼이 비활성화 상태라면 이동하지 않음
+      } else {
+          window.location.href = "/items";
+      }
+  });
+  // 이메일 조건문
+  function validateEmail() {
+      const emailValue = emailInput.value.trim();
+      const emailPattern = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+      removeErrorMessage(emailLabel);
+
+      if (emailValue === "") {
+          showError(emailLabel, "이메일을 입력해주세요.");
+          emailInput.classList.add("error");
+          return false;
+      } else if (!emailPattern.test(emailValue)) {
+          showError(emailLabel, "잘못된 이메일입니다.");
+          emailInput.classList.add("error");
+          return false;
+      } else {
+          emailInput.classList.remove("error");
+          return true;
+      }
```

Comment:
❗️ 수정요청
중간 중간에 return문을 사용하고 있으셔서 else if문을 쓰실 필요가 없어 보입니다~

```suggestion
      if (emailValue === "") {
          showError(emailLabel, "이메일을 입력해주세요.");
          emailInput.classList.add("error");
          return false;
      }
      
      if (!emailPattern.test(emailValue)) {
          showError(emailLabel, "잘못된 이메일입니다.");
          emailInput.classList.add("error");
          return false;
      }
      
      emailInput.classList.remove("error");
      return true;
```